Platforms/QemuQ35Pkg/PlatformBuild.py

```diff
@@ -493,9 +476,55 @@ def GetFileContent(self, VirtualFilePath, HostFilePath: os.PathLike=None):
             return f.read()
 
     def ExtractFile(self, VirtualFilePath, HostFilePath:os.PathLike):
-        ret = RunCmd("FileExtract", f"{VirtualFilePath} {HostFilePath} {self.path_to_vhd}")
+        ret = RunCmd("FileExtract", f"{VirtualFilePath} {HostFilePath} {self.drive_path}")
+        return ret
+
+class LinuxVirtualDriveManager(object):
+
+    def __init__(self, img_path:os.PathLike,  drive_letter:str="x"):

```

Comment:
I understand we want a default for the CI build, but is this potentially dangerous if a user has a drive mounted at X: already and didn't change the default? I.e. is it going to overwrite the drive? If so, perhaps some safety around this would be nice (or just a check if X: already exists and failing).

Reply:
Added a function find_unused_drive_letter() which will query the mtab file for mounted drives.